src/UnifiedFFI/FFILibrary.class.st

```diff
@@ -79,25 +81,27 @@ FFILibrary >> uniqueInstance [
 ]
 
 { #category : #'accessing platform' }
-FFILibrary >> unix32ModuleName [
```

Comment:
Is it possible to deprecate the method instead of removing it to ease migration of existing FFI bindings?

Reply:
This method is meant to be overridden. Users overriding it will still work as it is still called :) I did it in such a way that it is backwards compatibility.